### ISSUE TITLE
Update wiki page to dunst migration to work for Rofication

### ISSUE THREAD

**Issue by grzkv** [regolith-linux/regolith-desktop#279]
Looks like this guide
https://github.com/regolith-linux/regolith-desktop/wiki/HowTo:-Use-Dunst-to-view-notifications

does not work for the newer versions that use Rofication.

**Comment by grzkv** [comment]
I've made the changes in the wiki page, please review. Thanks!

**Comment by kgilmer** [comment]
Awesome, thanks for making the update @grzkv !  The wiki edit looks good.  It brings up an interesting point, if Xresources referenced the apps that are launched, users would not need to customize the i3 config file to change things like rofication -> dunst.  Rather they could just override the default in their own `~/.config/regolith/.Xresources`.  Making a user copy of i3 config is relatively disruptive as there are typically several changes from version to version and it's up to the user to merge their differences into the new upstream copy.

**Comment by grzkv** [comment]
> Making a user copy of i3 config is relatively disruptive as there are typically several changes from version to version and it's up to the user to merge their differences into the new upstream copy.

I completely agree with this point. For me, everything broke completely after major Regolith upgrade. After I removed my copies of settings, everything returned back to normal. So, I had to migrate all my settings manually. If we could resolve this, that would be awesome.

Regarding Rofication, I have to say that while I really like the idea of the system, I am not sure it is mature enough for general use. I like the idea of non-disruptive notifications, but the usability of it is far from perfect. I don't think it makes sense to go into details about that here. Also, it did not work for all the notifications for some reason. It worked for Slack, but did not work for Chrome notifications.
